doc/nrf/known_issues.rst

```diff
@@ -1285,6 +1285,26 @@ OCT-2154: USB audio interface does not work correctly on macOS
   The audio stream is intermittent on the headset side after connecting the gateway to a Mac computer and starting audio stream.
   This issue occurs sporadically after building the nRF5340 Audio application with the default USB port as the audio source.
 
+Controller - LE Audio subsystem for nRF5340:
+
+OCX-138: Some conformance tests not passing.
+
+OCX-155: Timestamps.
+   The timestamps (SDU ref) may occasionally be larger than intended and then duplicated in the next packet.
+
+OCX-156: PTO is not supported.
+
+OCX-157: Interleaved broadcasts
+   Does function for RTN > 2 and octets per frame > 80.
+
+OCX-168: Re-establishing streams
+   Reconnection of broadcast receivers takes longer than in version 3310, especially for high RTN (retransmit) values.
+
+If the first CIS headset in a CIG is reset, this may impact the stream of the second headset as well.
+
+If BIS and ACLs are used on the same broadcaster/central, the ACLs needs to be created first.
+
+
```

Comment:
```suggestion
Controller subsystem for nRF5340 Audio
--------------------------------------

The following known issues apply to the LE Audio subsystem for nRF5340 used in the nRF5340 Audio application.

.. rst-class:: v2-2-0

OCX-138: Some conformance tests not passing

.. rst-class:: v2-2-0

OCX-155: Larger timestamps than intended
   The timestamps (SDU ref) may occasionally be larger than intended and then duplicated in the next packet.

.. rst-class:: v2-2-0

OCX-156: PTO is not supported

.. rst-class:: v2-2-0

OCX-157: Interleaved broadcasts
   Does function for RTN > 2 and octets per frame > 80.

.. rst-class:: v2-2-0

OCX-168: Re-establishing streams
   Reconnection of broadcast receivers takes longer than in version 3310, especially for high RTN (retransmit) values.

If the first CIS headset in a CIG is reset, this may impact the stream of the second headset as well.

If BIS and ACLs are used on the same broadcaster/central, the ACLs needs to be created first.


```
Issues that go against the HOWTO guidelines in lines 56-71 of this file:

- OCX-138 needs fleshed out description, also to clarify the very generic "some" in the title.
- OCX-155 has unclear reference "(SDU ref)" -- please make it more readable.
- OCX-156 needs fleshed out description, also to clarify the PTO acronym in the title.
- OCX-157 needs clearer title and description.
- OCX-168 needs clearer title and description.
- The last two lines are not clear -- are these issues or continuation of OCX-168? If new issues, they need a JIRA and title + description.

Does any of the entries have a workaround?